homeassistant/components/opentherm_gw/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/opentherm_gw",
   "iot_class": "local_push",
   "loggers": ["pyotgw"],
-  "requirements": ["pyotgw==2.1.3"]
+  "requirements": ["pyotgw==2.2.0"]
```

Comment:
Should the dependency update be split into a separate PR?

Reply:
This PR has a dependency on pyotgw 2.2.0. How is this normally handled?

Reply:
As I remember it, it is customary to update the requirements first before adding new features that depend on them. See also the [development checklist](https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible), especially item 5.

Reply:
Ok clear, will create a bump PR for pyotgw 2.2.0

Reply:
When you do, please specify that it fixes  #113443

Reply:
Done, I also reverted the requirement change within this PR.